R/print.R

```diff
@@ -21,6 +28,13 @@ knit_print.plotly_hash <- function(x, options, ...) {
   knitr::knit_print(x, options, ...)
 }
 
+#' Print a 'built' plotly object in a knitr doc
+#' 
+#' @param x a plotly object
+#' @param ... additional arguments
+#' @export
+knit_print.plotly_built <- knit_print.plotly_hash
+
```

Comment:
@jcheng5 I've been meaning to distinguish between "non-built" (e.g., `plot_ly()`) and "built" (e.g., `plotly_build(plot_ly())` plotly objects for a while now. I haven't needed it up until this point since `plotly_build()` is idempotent (i.e., building a plot more than once should be the same as building it once). That being said, going forward I think it's a good idea to explicitly distinguish between these two types of objects.